### ISSUE TITLE
Make next piece sounds understandable

### ISSUE THREAD

**Issue by Nathan-Huckleberry** [tetrio/issues#623]
### Checklist (issue will be disregarded & closed if incomplete!)

- [X] I've **checked both [open and recently closed issues](https://github.com/tetrio/issues/issues?q=is%3Aissue)** to ensure this feature is not already requested.
- [X] I am using the **newest version** of TETR.IO.
- [X] I am certain this is a **feature request**, not a bug.
- [X] I've **thought this feature through** properly.

-----

### Is your feature request related to a problem? Please describe.
The next piece sounds are very hard to distinguish. I occasionally practice blindfolded and I have to manually swap in an edited sound file with a MITM proxy. When Tetrio updates, I have to make a new sound file which is annoying.

Side note, but it'd also be nice to create a keybind to replay the current/next piece sounds. I can create another feature request if you think it's worthwhile.

### Describe the solution you'd like
A setting to change the next piece sounds to understandable versions would be nice.

### Describe alternatives you've considered
I've been using a MITM proxy and it's annoying.

### Additional context
I've attached the audio files that I swap in. These came from Google TTS https://cloud.google.com/text-to-speech.
[sounds.zip](https://github.com/tetrio/issues/files/5939984/sounds.zip)

**Comment by ArmandoRl1** [comment]
Yeah, the current sounds are slow/hard to recognize and I'd like to change them.
